src/docs/content/observability/observability-example/usecase.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Use Case"
+date: 2020-03-10T13:29:23-06:00
+draft: false
+tags:
+  - observability in action
+  - observability use case
+  - observability example
+  - MuShop Payment Failures
```

Comment:
I don't feel that these are an appropriate use of tags. Instead, simply list any relevant services or features here. Like:
- Observability
- Custom Logging
- Monitoring
- Metrics Explorer
- ...